sections/04_study.md

```diff
@@ -52,7 +52,104 @@ particularly notable in this area?*
 
 ### Protein secondary and tertiary structure
 
-*Jinbo Xu is writing this*
+Proteins play fundamental roles in all biological processes including the 
+maintenance of cellular integrity, metabolism, transcription/translation, and 
+cell-cell communication. Complete description of protein structures and 
+functions is a fundamental step towards understanding biological life and 
+also highly relevant in the development of therapeutics and drugs. Tons of 
+protein sequences have been generated, but fewer than 100,000 of them 
+have experimentally-solved structures. As a result, computational structure 
+prediction is essential for a majority number of protein sequences. However, 
+predicting protein 3D structures from sequence alone is very challenging, 
+especially when similar templates are not available. In the past decades, 
+various computational methods have been developed to predict protein 
+structure from different aspects, including prediction of secondary structure, 
+torsion angles, solvent accessibility, inter-residue contact map, disorder 
+regions and side-chain packing.
+
+Machine learning is extensively applied to predict protein structures and 
+some success has been achieved. For example, secondary structure can be 
+predicted with about 80% of Q3 accuracy by a 2-layer neural network 
+method PSIPRED [@doi:10.1093/bioinformatics/16.4.404]. Starting from 
+2012, deep learning has been gradually introduced to protein structure 
+prediction. The adopted deep learning models include deep belief network, 
+LSTM(long short-term memory), deep convolutional neural networks (DCNN) 
+and deep convolutional neural fields[@doi:10.1007/978-3-319-46227-1_1 
+@doi:10.1038/srep18962]. Here we focus on deep learning methods for 
+two representative subproblems: secondary structure prediction and 
```

Comment:
I'm strongly in favor of limiting scope and focusing on protein structure-related problems where 1) we have something to say instead of just enumerating methods and 2) deep learning is  contributing something unique, so this sounds great to me.  It could be helpful to bring some of the context in your comment into the manuscript.  Something like:

> Starting from 2012, deep learning has been gradually introduced to multiple aspects of protein structure prediction ranging from **X** [ref] to **Y** [ref] to **Z** [ref].  The adopted deep learning models include deep belief network, LSTM(long short-term memory), deep convolutional neural networks (DCNN) and deep convolutional neural fields[@doi:10.1007/978-3-319-46227-1_1 @doi:10.1038/srep18962]. Here we focus on two representative subproblems where deep learning has been especially impactful: secondary structure prediction and contact map prediction.

I don't necessarily like the phrasing I suggested, but the idea would be to show that we are explicitly narrowing our scope, calling out to a few other deep learning methods in other areas of structure prediction that might be interesting but are out of that scope, and stating why we focused on the sub-problem we did.